### ISSUE TITLE
make update_caravel fails

### ISSUE THREAD

**Issue by buzmeg** [efabless/caravel_user_project_analog#28]
```make update_caravel
cd /home/foo/efabless/caravel_user_project_analog/caravel/ && git checkout mpw-7a && git pull
HEAD is now at de98d51 Modified the set_user_id.py script so that mode "-report" returns a valid value. (#93)
You are not currently on a branch.
Please specify which branch you want to merge with.
See git-pull(1) for details.

    git pull <remote> <branch>

make: *** [Makefile:92: update_caravel] Error 1
```

**Comment by d-m-bailey** [comment]
This is called because the current tag is a detached HEAD and not on a branch.

Fixed with PR #30

**Comment by d-m-bailey** [comment]
@SaraEfabless This will be fixed with PR #30

Probably want to do the same thing with `caravel_user_project`. Should I create a PR there, too?

**Comment by SaraEfabless** [comment]
Hi Mitch,
Thanks for your fast response! No, I don’t think if it needs another PR. I’m going to close this ticket with adding comments. 

Sara

**Comment by d-m-bailey** [comment]
@SaraEfabless I think the the pull request should be merged before this issue is closed.
